esp_driver_sccb/sccb_i2c/include/sccb_i2c.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "driver/i2c_master.h"
```

Comment:
what IDF version do we expect this component depends on? FYI, i2c_master.h was introduced since v5.2, so if we allow the user to use sccb on other idf version, we need to provide another implemnetation

Reply:
Added `idf_component.yml`, is it correct?

Reply:
I think we don't allow users using sccb for versions previous than v5.2

Reply:
> Added idf_component.yml, is it correct?

yes. 

> I think we don't allow users using sccb for versions previous than v5.2

Not sure if esp_video wants to support other IDF releases. Never mind ATM, we can always add legacy driver compatible layer later.